Sources/KnitCodeGen/FunctionCallRegistrationParsing.swift

```diff
@@ -271,6 +277,16 @@ private func getArguments(
     return []
 }
 
+private func getConcurrencyModifier(trailingClosure: ClosureExprSyntax?) -> String? {
+    guard let signature = trailingClosure?.signature else { return nil }
+    for att in signature.attributes {
+        if att.description.trimmingCharacters(in: .whitespaces) == "@MainActor" {
+            return "@MainActor"
```

Comment:
I'd expect there is already a token for this within swift-syntax?

Reply:
There's nothing in `TokenKind`, I'm not sure where else it might be.

Reply:
I took a look too and couldn't find anything, oh well